### PR TITLE
hotifx/removing warning logs

### PR DESCRIPTION
**Pull request by Affeeq**
- removed as many react warning logs
- removed comments

### CODE REVIEW

File: client/src/components/Editor/DraftEditor.js [hunk 1]
```diff
@@ -18,31 +18,8 @@ import { makeStyles } from "@material-ui/core/styles";
...
-  clike,
-  ruby,
-  php,
-  python,
-  c,
-  cpp,
-  csharp,
-  java,
-  languagesGrammar,
-} from "./languages";
```
Comment (svitaly) [inline client/src/components/Editor/DraftEditor.js:32]: I don't think that without languages we will have prism highlighting. Have u tested it?

**Review by svitaly — COMMENTED** [review]
I'm concerned about languages in DraftEditor.js. Without import highlighting doesn't work. And console.log was written to remove warning of not used variables

File: client/src/components/Editor/DraftEditor.js (hunk 1 above)
Comment (Affeeq) [reply client/src/components/Editor/DraftEditor.js:32]: yes prism editor still has highlighting after I remove the languages import. None of the languages were used in the file, thus the reason why I removed it.

**Comment by LiveBacteria** [comment]
If the draftEditor has been resolved, everything looks good here.

**Review by svitaly — APPROVED** [review]

**Pull request merged**